plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -114,15 +128,17 @@
     description:
       - Allow to force stop VM.
       - Can be used with states C(stopped), C(restarted) and C(absent).
-    default: no
```

Comment:
Shouldn't `force:` get the other descriptions as well?

Reply:
Yes, thanks for mentioning. Fixed in https://github.com/ansible-collections/community.general/pull/850/commits/4e19b4b03415a80f4e4ab3c1ecded324b68fd7f4.